src/BoundaryConditions/BoundaryConditions.jl

```diff
@@ -4,20 +4,23 @@ export
     BCType, Flux, Gradient, Value, Open,
     BoundaryCondition, getbc, setbc!,
     PeriodicBoundaryCondition, OpenBoundaryCondition, NoFluxBoundaryCondition, MultiRegionCommunicationBoundaryCondition, 
+    MeanOutflowOpenBoundaryCondition,
```

Comment:
I don't see `MeanOutflow` anywhere else. Is this leftover code?

Reply:
Ah yes this is leftover

```suggestion
```